packages/aragon-wrapper/src/index.js

```diff
@@ -278,17 +300,19 @@ export default class Aragon {
     this.identifiers = new Subject()
     this.appsWithoutIdentifiers = this.permissions
       .map(Object.keys)
-      .map((addresses) =>
-        addresses.filter((address) => !addressesEqual(address, this.kernelProxy.address))
-      )
+      // Add Kernel as the first "app"
+      .map((addresses) => {
+        const appsWithoutKernel =
+          addresses.filter((address) => !addressesEqual(address, this.kernelProxy.address))
+        return [this.kernelProxy.address].concat(appsWithoutKernel)
+      })
+      // Get proxy values
       .switchMap(
         (appAddresses) => Promise.all(
-          appAddresses.map((app) => this.getAppProxyValues(app))
+          appAddresses.map((app) => this.getProxyValues(app))
         )
       )
-      .map(
-        (appMetadata) => appMetadata.filter((app) => this.isApp(app))
```

Comment:
Is the `isApp` filter removed? Can't find where it is being checked right now.

This check is quite important, otherwise apps from other DAOs that just happen to be mentioned in the ACL will show up as DAO apps.